Modules/Product/Http/Services/ProductService.php

```diff
@@ -0,0 +1,1184 @@
+<?php
+
+namespace Modules\Product\Http\Services;
+
+use Modules\Product\Entities\Product;
+use Illuminate\Support\Str;
+use Modules\Product\Entities\Video;
+use Modules\Product\Entities\ProductVariation;
+use Modules\Product\Entities\ProductImage;
+use Modules\Product\Entities\ProductPrepack;
+use Modules\Product\Entities\Category;
+use File;
```

Comment:
Use facade instead

Reply:
Can you describe the reason for not using "use" statement and using of Facade...

Reply:
Those are old versions, for new version we are using
Illuminate\Support\Facades\DB;
Illuminate\Support\Facades\File;

Even with PHP storm if you do alt+enter it will use Facade

Reply:
Added Illuminate\Support\ version in productservice.. Please check